scripts/tests/scripts_regression_tests.py

```diff
@@ -1399,6 +1403,29 @@ def test_cime_case(self):
             self.assertEqual(case.get_value("TESTCASE"), "TESTRUNPASS")
 
     ###########################################################################
+    def test_cime_case_prereq(self):
```

Comment:
I'm somewhat concerned that this would depend on the queue system not fortuitously scheduling the jobs regardless of whether the prerequisites have been met.
The other issue is it's not clear to me how to get job 1's queue id? run_indv calls case_run, which assumes it's already on the queue.